ext/SparseMatrixColoringsCliqueTreesExt.jl

```diff
@@ -0,0 +1,36 @@
+module SparseMatrixColoringsCliqueTreesExt
+
+using CliqueTrees: CliqueTrees
```

Comment:
You can import everything at the top, it makes it easier to see what we depend on from CliqueTrees